scaleway/resource_lb.go

```diff
@@ -201,24 +199,23 @@ func resourceScalewayLbRead(ctx context.Context, d *schema.ResourceData, meta in
 	_ = d.Set("region", region.String())
 	_ = d.Set("organization_id", lb.OrganizationID)
 	_ = d.Set("project_id", lb.ProjectID)
-	_ = d.Set("tags", lb.Tags)
+	if len(lb.Tags) > 0 {
```

Comment:
Why does this line need to be changed?

Reply:
This line is causing a diff after a resource creation because the attribute is `Optional` and the API is returning `[]`. As result the sdk is asking an extra `apply`.

Reply:
What happens in the case where the tags are deleted and now there are 0 tags in the list but the state is not yet updated?